apps/examples/err_report_demo/err_report_main.c

```diff
@@ -295,17 +295,35 @@ void error_report_process(int argc, char *argv[])
 		printf("%s", ERR_REPORT_USAGE);
 		return;
 	}
-	if (strcmp(argv[2], "send") == 0) {
+	if (strcmp(argv[2], "test") == 0) {
 		if (argc < 4) {
-			error_report_records(NULL);
+			error_report_test(NULL);
 		} else {
-			error_report_records(argv[3]);
+			error_report_test(argv[3]);
 		}
-	} else if (strcmp(argv[2], "test") == 0) {
+	} else if (strcmp(argv[2], "report_send") == 0) {
 		if (argc < 4) {
-			error_report_test(NULL);
+			error_report_records(NULL);
 		} else {
-			error_report_test(argv[3]);
+			error_report_records(argv[3]);
+		}
+	} else if (strcmp(argv[2], "report_print")) {
+		int nbytes_read;
+		char readbuf[1024];
```

Comment:
You could use #define for defining buffer size

Reply:
Thanks for the comment. I have modified as suggested.